cli/Cargo.toml

```diff
@@ -13,8 +13,9 @@
 # limitations under the License.
 
 [features]
-experimental = ["health"]
+experimental = ["health", "circuit"]
```

Comment:
Please add a newline below experimental and then sort the remainder of the features.

Reply:
done